versions/3.0.md

```diff
@@ -2323,7 +2327,9 @@ While composition offers model extensibility, it does not imply a hierarchy betw
 To support polymorphism, OpenAPI Specification adds the support of the `discriminator` field.
 When used, the `discriminator` will be the name of the property used to decide which schema definition is used to validate the structure of the model.
 As such, the `discriminator` field MUST be a required field.
-The value of the chosen property has to be the friendly name given to the model under the `definitions` property.
+There are are two ways to define the value of a discriminator for an inheriting instance.
```

Comment:
Good point -- we rely heavily on external schema files via `$ref`. For example, if Cat and Dog are defined externally, I should still be able to use them. However for a schema included from an external schema.json there is no name as there is for one defined in `/components/definitions`. If all three schema (Pet, Cat, Doc) were defined externally (so that they can be used in multiple OpenAPI documents) and there is no single `/components/definitions` in which they reside, how would this work?